backend/src/game_objects/hex.rs

```diff
@@ -0,0 +1,64 @@
+use super::content::Wall;
+use super::unit::Unit;
+use serde::Serialize;
+
+#[derive(Clone, Serialize)]
+pub struct Hex {
+    pub x: u32,
+    pub y: u32,
+    pub unit: Option<Unit>,
+    pub content: Option<Wall>,
```

Comment:
I think that if variable has type `Option<Wall>`, then it seems only logical for it to be named `wall`.
If later we are to change the type, we may also change the variable name at the same time.

Reply:
changed as we discussed